src/java/org/apache/cassandra/index/sai/plan/QueryController.java

```diff
@@ -512,7 +518,26 @@ private Map<Expression, NavigableSet<SSTableIndex>> referenceAndGetView(Operatio
         }
         finally
         {
-            Tracing.trace("Querying storage-attached indexes {}", indexNames);
+            traceGroupedIndexes(referencedIndexes, queryContext);
```

Comment:
does this make sense to run even if there's an exception?  if not it should not be in finally block

Reply:
I'll refactor it, getView seems to be the only place that can throw, and in this case tracing isn't helpful anyway